### PR TITLE
update image code and include caption

### PR DESCRIPTION
**Pull request by bonnieAcameron**
Hi Amy! I can't access federalist, can you take a look and see if this looks right?

Preview link: [Maturity model page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/bc-add-captions/maturity-model/)

### CODE REVIEW

**Comment by amyleadem** [comment]
@bonnieAcameron Looks good! I found an extra `</div>` tag so I pushed up a couple of changes while I was in there:
- Formatted the code
- Created some styles for `<figure>` and `<figcaption>`

**Comment by bonnieAcameron** [comment]
Thank you, @amyleadem! It looks nice. Is this ready to be approved?

**Comment by bonnieAcameron** [comment]
Hey @amyleadem! Can you approve this if all looks good?

File: pages/documentation/maturity-model.md [hunk 1]
```diff
@@ -29,11 +29,10 @@ subnav:
...
       </div>
     </div>
-    <div class="tablet:grid-col-5">
-      <div class="padding-2 tablet:padding-left-4">
-        <img src="{{ site.baseurl }}/img/maturity-model.svg" role="img" alt="A diagram showing the concentric circles of the USWDS maturity model, progressing from Principles on the outside, through Guidance, and finally to Code in the innermost circle.">
-      </div>
-    </div>
+    <figure class="site-figure tablet:grid-col-5 padding-2 tablet:padding-left-4">
+      <img src="{{ site.baseurl }}/img/maturity-model.svg" role="img" alt="A diagram showing the concentric circles of the USWDS maturity model, progressing from Principles on the outside, through Guidance, and finally to Code in the innermost circle.">
+      <figcaption class="site-figcaption">In this image you can see 3 levels of "maturity" in relation to the U.S. Web Design System. As you can see, you don't have to adopt the code to adopt the guidance and benefits outlined by the Design System</figcaption>
```
Comment (amyleadem) [inline pages/documentation/maturity-model.md:34]: Should the `3` be  spelled out here? Relying on my college AP style class here, so let me know if that isn't what we do here :disguised_face:

Also, found a missing a period at the end. 

```suggestion
      <figcaption class="site-figcaption">In this image you can see three levels of "maturity" in relation to the U.S. Web Design System. As you can see, you don't have to adopt the code to adopt the guidance and benefits outlined by the Design System.</figcaption>
```

File: css/_uswds-theme-custom-styles.scss [hunk 2]
```diff
@@ -519,6 +519,19 @@ img {
...
+  @include u-flex("align-center");
+  @include u-margin(0);
+}
+
+.site-figcaption {
+  @include u-font-size("lang", "2xs");
+  @include u-margin-y(2);
+  color: color("gray-60");
+}
+
```
Comment (amyleadem) [inline css/_uswds-theme-custom-styles.scss:534]: @mejiaj These might be my first styles created for site. Can you review these updates and let me know if they follow our standards?

**Comment by amyleadem** [comment]
@bonnieAcameron Just looked again and left a copy comment. Also tagged @mejiaj for review since this is adding some new styling to site and want to confirm that the code matches standards.

File: pages/documentation/maturity-model.md [hunk 3]
```diff
@@ -29,11 +29,10 @@ subnav:
...
       </div>
     </div>
-    <div class="tablet:grid-col-5">
-      <div class="padding-2 tablet:padding-left-4">
-        <img src="{{ site.baseurl }}/img/maturity-model.svg" role="img" alt="A diagram showing the concentric circles of the USWDS maturity model, progressing from Principles on the outside, through Guidance, and finally to Code in the innermost circle.">
-      </div>
-    </div>
+    <figure class="site-figure tablet:grid-col-5 padding-2 tablet:padding-left-4">
+      <img src="{{ site.baseurl }}/img/maturity-model.svg" role="img" alt="A diagram showing the concentric circles of the USWDS maturity model, progressing from Principles on the outside, through Guidance, and finally to Code in the innermost circle.">
+      <figcaption class="site-figcaption">In this image you can see three levels of "maturity" in relation to the U.S. Web Design System. As you can see, you don't have to adopt the code to adopt the guidance and benefits outlined by the Design System.</figcaption>
```
Comment (thisisdano) [inline pages/documentation/maturity-model.md:34]:
```suggestion
      <figcaption class="site-figcaption">The USWDS maturity model shows how teams can grow their USWDS maturity by starting with Principles (the outermost circle) and progressing through Guidance (the middle circle) and Code (the innermost circle).</figcaption>
```

File: css/_uswds-theme-custom-styles.scss [hunk 4]
```diff
@@ -519,6 +519,19 @@ img {
   }
 }
 
+.site-figure {
+  @include u-display("flex");
+  @include u-flex("column");
+  @include u-flex("align-center");
```
Comment (mejiaj) [inline css/_uswds-theme-custom-styles.scss:525]: These styles follow standards, but I don't see a visual difference with these disabled. Do we need them?

File: css/_uswds-theme-custom-styles.scss [hunk 5]
```diff
@@ -519,6 +519,19 @@ img {
...
+  @include u-margin(0);
+}
+
+.site-figcaption {
+  @include u-font-size("lang", "2xs");
+  @include u-margin-y(2);
+  color: color("gray-60");
```
Comment (mejiaj) [inline css/_uswds-theme-custom-styles.scss:532]: Color is set via `color: color($site-ink);`. 

Alternatively, we can remove and it should inherit the correct color.

**Review by mejiaj — DISMISSED** [review]
@amyleadem reviewed styles and added comments. Otherwise PR looks good.

File: css/_uswds-theme-custom-styles.scss (hunk 2 above)
Comment (mejiaj) [reply css/_uswds-theme-custom-styles.scss:534]: Sure, added my review below. (https://github.com/uswds/uswds-site/pull/1751#pullrequestreview-1095073705)

File: css/_uswds-theme-custom-styles.scss (hunk 5 above)
Comment (amyleadem) [reply css/_uswds-theme-custom-styles.scss]: Thanks! Fixed in [ba6df02](https://github.com/uswds/uswds-site/pull/1751/commits/ba6df02ca7f5de9e0b88c84dc1cb34aac64746ad)

File: css/_uswds-theme-custom-styles.scss (hunk 4 above)
Comment (amyleadem) [reply css/_uswds-theme-custom-styles.scss]: Removed in [1c0a212](https://github.com/uswds/uswds-site/pull/1751/commits/1c0a212669a3082bcd6fadae7ad5d0dd453fb859)

**Review by amyleadem — APPROVED** [review]
Everything looks good to me. 

@mejiaj can you take another look?

**Review by thisisdano — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**